### DISCUSSION TITLE
Ubuntu Release版本编译失败规避方法

### DISCUSSION THREAD

**Discussion by robinhzp** [alibaba/xquic#49] [Show and tell]
set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O2 -std=gnu11 **-Wno-stringop-overflow** -Werror -Wall -Wno-unused -Wno-pointer-sign -Wno-format-security -DNDEBUG_PRINT -DNPRINT_MALLOC ")

增加-Wno-stringop-overflow 选项

**Comment by quietUncle** [comment]
👍🏻,有用
